.github/workflows/publish-docker-e2e-service.yaml

```diff
@@ -1,36 +1,19 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
we should not remove any licensed or reference to open source

Reply:
okay !